### PR TITLE
Create version-11-color.md

### LINKED ISSUE #8726: Add "What's New" post for new color tokens
Post "What's new" post on color changes to polaris.shopify.com

### PR DESCRIPTION
**Pull request by heyjoethomas**
### WHY are these changes introduced?
Closes: #8726

With the update in our color palette and token package, this post is part of the comms strategy for sharing our strategy behind the changes made.

### WHAT is this pull request doing?
This PR is creating a new blog post on the What's New section of the site. 

### How to 🎩

🖥 [Local development instructions](https://github.com/Shopify/polaris/blob/main/README.md#local-development)
🗒 [General tophatting guidelines](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md)
📄 [Changelog guidelines](https://github.com/Shopify/polaris/blob/main/.github/CONTRIBUTING.md#changelog)

<details>
<summary>Copy-paste this code in <code>playground/Playground.tsx</code>:</summary>

```jsx
import React from 'react';
import {Page} from '../src';

export function Playground() {
  return (
    <Page title="Playground">
      {/* Add the code you want to test in here */}
    </Page>
  );
}
```

</details>

### 🎩 checklist

- [ ] Tested on [mobile](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting.md#cross-browser-testing)
- [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/shopify-admin/supported-browsers)
- [ ] Tested for [accessibility](https://github.com/Shopify/polaris/blob/main/documentation/Accessibility%20testing.md)
- [ ] Updated the component's `README.md` with documentation changes
- [ ] [Tophatted documentation](https://github.com/Shopify/polaris/blob/main/documentation/Tophatting%20documentation.md) changes in the style guide

### CODE REVIEW

File: .changeset/rude-teachers-doubt.md
Comment (laurkim) [inline .changeset/rude-teachers-doubt.md]:
```suggestion
Added what's new color page and updated design/colors for the new color tokens
```

**Review by laurkim — APPROVED** [review]
Nice! 💯 

Should this be pointing to the `v11-major` branch? Or is it intent to merge this after the v11 branch merges to `main`?

**Comment by heyjoethomas** [comment]
> Nice! 💯
> 
> Should this be pointing to the `v11-major` branch? Or is it intent to merge this after the v11 branch merges to `main`?

I think the intent was to have it go out with 11. But since it is already released it could really go out now tbh.

**Review by chloerice — APPROVED** [review]
🙌🏽 🚢

**Review by chloerice — APPROVED** [review]
This looks great @heyjoethomas!! Realized while tophatting the site that  we forgot to add a gap to the grid on the What's new page (since there's only one currently, it wasn't noticable until now). Can you add `gap: 1.5rem` (or whatever you think is best!) to [this class name](https://github.com/Shopify/polaris/blob/8e6628556863dd1e723e716041642e24ed570356/polaris.shopify.com/src/components/WhatsNewListing/WhatsNewListing.module.scss#L4)?

| Before | After |
|--------|--------|
|Screenshot 2023-04-14 at 3 27 36 PM| Screenshot 2023-04-14 at 3 27 26 PM |

**Comment by heyjoethomas** [comment]
> This looks great @heyjoethomas!! Realized while tophatting the site that we forgot to add a gap to the grid on the What's new page (since there's only one currently, it wasn't noticable until now). Can you add `gap: 1.5rem` (or whatever you think is best!) to [this class name](https://github.com/Shopify/polaris/blob/8e6628556863dd1e723e716041642e24ed570356/polaris.shopify.com/src/components/WhatsNewListing/WhatsNewListing.module.scss#L4)?
> 
> Before	After
> Screenshot 2023-04-14 at 3 27 36 PM	Screenshot 2023-04-14 at 3 27 26 PM

Done. Thanks Chloe

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [inline polaris.shopify.com/content/whats-new/version-11-color.md:14]: This paragraph might not be needed. It's a lot of not specific information and the real value in "Why are we making these changes" is in the dot points below.

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [inline polaris.shopify.com/content/whats-new/version-11-color.md]:
```suggestion
1. Create a defined color palette with multiple lightness values that is used to create new tokens. This will eliminate color sprawl and ensure a more consistent and cohesive use of color across the admin.
```

We don't generate any tokens in our system. They are all chosen with strong design opinions. We use the word generate a bunch and from what I can tell they are all chosen colors that are not generated by any code or algorithms. Maybe we can use a different word? Please correct me if I am wrong.

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [inline polaris.shopify.com/content/whats-new/version-11-color.md:21]: The ordered list of these points is a little bit confusing to me. 2/3/4 seem a lot more important to me then 1. Maybe they should just be an unordered list?

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [inline polaris.shopify.com/content/whats-new/version-11-color.md]:
```suggestion
We started with an audit of the colors used in Polaris. Previously we didn’t have a defined color palette to create tokens from. This led to massive sprawl and inconsistency of hue, saturation, and vibrancy within our colors. We learned that there were 22 different gray values in Polaris, some of which were nearly identical to each other, providing no value while introducing added complexity. Every hue had a different range of tints and shades with no consistency or relationship between them.
```

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [inline polaris.shopify.com/content/whats-new/version-11-color.md]:
```suggestion
We took the previous colors and created a new color palette with 10 shades for each hue. Each shade should be meaningfully different from the ones next to it on the scale. This makes choices purposeful and clear and eliminates the sprawl and muddiness we had before.
```

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [inline polaris.shopify.com/content/whats-new/version-11-color.md]:
```suggestion
These colors power every token provided within the admin. This means that we can update a single color value within our palette and it will cascade down to every token referencing it, updating the entire admin from a single and simple change.
```

"for the first time" makes sense and also doesn't... We could change colors through our tokens and now we are saying we can change colors from the color palette into the tokens into the admin. Maybe there is a way to frame this so it sounds like a win but also doesn't make it sound like we couldn't make color changes at scale.

...I might be reading into this too much though and this could be fine.

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [inline polaris.shopify.com/content/whats-new/version-11-color.md]:
```suggestion
There are now two types of color tokens in Polaris. The palette tokens which are private and used to create new aliases and the alias tokens which is what designers and developers use.
```

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [inline polaris.shopify.com/content/whats-new/version-11-color.md:39]: I am getting stuck on this framing that the color palette is not for use... If it truely wasn't for use it would be completely private and we wouldn't talk about it. The color palette is available to be used and is documented, however there is only one way to use it and it is for creating new alias tokens.

**Review by alex-page — APPROVED** [review]
This is really good. I added some comments but I would also be happy if this shipped as is.

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (heyjoethomas) [reply polaris.shopify.com/content/whats-new/version-11-color.md:14]: I think this is nice contextual info. I get your point, but think it helps tie this work in with all the other stuff we have shipped.

**Review by aaronccasanova — APPROVED** [review]

File: polaris.shopify.com/content/whats-new/version-11-color.md
Comment (alex-page) [reply polaris.shopify.com/content/whats-new/version-11-color.md:14]: Makes sense to me @heyjoethomas 👍

**Review by chloerice — APPROVED** [review]
🌈 🚀

**Pull request merged**
